compile.ts

```diff
@@ -603,7 +603,7 @@ class Compiler {
     }
     this.#emit(info, ...args);
     dest.exec?.forEach((ref) => {
-      this.#rewrite(ref, "false", true);
```

Comment:
This doesn't seem right. `false` is like no connection in the in-game editor, and `nil` is a link to the next node in order. Why should these default to the next instruction?

Reply:
I was not aware of the difference between "false" and "nil". Let me rework on this. The current code has definitively some issue with "nil" or "false" that do not generate the right code.

Reply:
So looking into this, we should never use nil. The issue with nil is that it only makes sense if next is also the same and also use nil, otherwise the next instruction will be a jump and will defeat the purpose. So, instead of using these marker ("false" and "nil"), I will just use the null value.